meta-balena-common/recipes-core/initrdscripts/files/resindataexpander

```diff
@@ -74,6 +73,28 @@ resindataexpander_run() {
         fi
     done
 
+    # Expand dm-crypt device
+    if is_part_encrypted "${datapartdev}"; then
```

Comment:
This feels more branched than it needs to be, can we maybe do
```
if is_part_encrypted "${datapartdev}" && ! is_luks "${datapartdev}"
  _label=...
  if [ -z "${_label}" ]
    error
  fi

  _size=...
fi
```

Reply:
right, I'll simplify it, thanks